src/SFA.DAS.AssessorService.Application.Api/Services/StandardService.cs

```
@@ -84,9 +89,12 @@ public async Task<Standard> GetStandardVersionById(string id, string version = n
                     case StandardId.StandardIdType.StandardUId:
                         standard = await _standardRepository.GetStandardVersionByStandardUId(standardId.StandardUId);
                         break;
-                    default:
-                        throw new ArgumentOutOfRangeException("id", "StandardId was not of type StandardUId, LarsCode or IfateReferenceNumber");
-
+                    case StandardId.StandardIdType.Unknown:
```

This should still throw an exception here when the type is unknown